erpnext/support/doctype/issue/issue.json

```diff
@@ -1303,7 +1303,7 @@
  "sort_order": "ASC", 
  "timeline_field": "customer", 
  "title_field": "subject", 
- "track_changes": 0, 
+ "track_changes": 1, 
```

Comment:
No need to change this on the doctype level. You track changes by using ```customize form``` in **Menu Dropdown**
![Screenshot 2019-07-11 at 6 53 26 PM](https://user-images.githubusercontent.com/32095923/61054515-38717300-a40d-11e9-9423-a7f5090e98c3.png)